File: java/Semestre 2/Clase 1/Practica.java
// Ciclo while
var conteo = 0;
while (conteo < 6){
    System.out.println("conteo = " + conteo);
    conteo++;
}

// Ciclo do while
var contador = 0;
do{
    System.out.println("contador = " + contador)
    contador++;
}while(contador <  6);

// Ciclo for
for (int i = 0; i < 6; i++){
    System.out.println("contador del ciclo for = " + i);
}

// Palabra reservada break
for (int i = 0; i < 6; i++{
    if (i % 2 == 0){
        System.out.println("ciclo for = " + i);
        break;
    }
}

// Palabra reservada continue
for (int i = 0; i < 6; i++{
    if (i % 2 != 0){
        continue;
    }
    System.out.println("ciclo for = " + i);
}

// Etiquetas Labels
inicio:
for (int i = 0; i < 6; i++{
    if (i % 2 == 0){
        System.out.println("ciclo for = " + i);
        break inicio;
    }
}

File: javascript/Semestre 2/Clase 1/Practica.js
// Cicllos

// While
let contador = 0;
while (contador < 3) {
    console.log(contador);
    contador++;
}
console.log("Fin del ciclo while");

// Do While
let contador2 = 0;
do{
    console.log(contador2);
    contador2++;
}while (contador2 < 3) 
console.log("Fin del ciclo do while");

// For
for (let i = 0; i < 3; i++) { // el i seria el contador  
    console.log(i); // i++ es lo mismo que i = i + 1
}
console.log("Fin del ciclo for");

// Palabra reservada break
for(let j = 1; j <= 10; j++){
    if (j % 2 == 0) {
        console.log(j); 
        break; // rompe la estructura al encontrar el primer numero par
    }
}
console.log("Fin del ciclo al encontrar el primer numero par");
// otro ejemplo
let k = 3;
while (k > -3){
    k--;
    if (k < 0) {
        console.log(k);
        break; // rompe la estructura al encontrar el primer numero negativo
    }
}
console.log("Fin del ciclo al encontrar el primer numero negativo");

// Palabra reservada continue y Etiquetas Labels
etiqueta:
for (let l = 0; l <= 10; l++){
    if (l % 2 !== 0) {
        continue etiqueta;
    }
    console.log(l); // imprime los numeros impares
}
console.log("Fin del ciclo al");